module/dkg/broker_test.go

```diff
@@ -177,80 +176,14 @@ func TestReceivePrivateMessage_Valid(t *testing.T) {
 	// simulate receiving an incoming message through the broker
 	receiver.tunnel.SendIn(
 		msg.PrivDKGMessageIn{
-			DKGMessage: expectedMsg,
+			DKGMessage: dkgMessage,
 			OriginID:   committee[orig].NodeID,
 		},
 	)
 
 	unittest.RequireCloseBefore(t, doneCh, 50*time.Millisecond, "message not received")
 }
 
-// TestProcessPrivateMessage_InvalidOrigin checks that incoming DKG messages are
-// discarded if their origin is invalid, or if there is a discrepancy between
-// the origin defined in the message, and the network identifier of the origin
-// (as provided by the network utilities).
-func TestProcessPrivateMessage_InvalidOrigin(t *testing.T) {
```

Comment:
👍 for removing the case testing origin ID / origin index mismatch. But I think we should still have a test case for messages from a non-committee member.

Reply:
https://github.com/onflow/flow-go/pull/2238/commits/4ecc851cf54089b33217d7479e3bdaeeaab358cc